sagemaker-training-compiler/huggingface/pytorch_tune_batch_size/batch_tune.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>It seems that the launch_*.py scripts are not used anywhere. Please remove the files to avoid confusion. Those scripts are already shown in the other example notebooks, and keeping them here is not necessary. </p><p> </p><p>And change the pygmentize command to:</p><pre class="ql-syntax" spellcheck="false">!pygmentize ./scripts/find_max_batch_size.py
</pre><p> </p><p>Also please add more description about what this script does.</p><p> </p><p>Or, if those scripts are used somewhere, please explain where those are used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>